apps/back-office/src/app/(dashboard)/calendar/components/calendar/calendar.tsx

```
@@ -123,24 +304,25 @@ export function CalendarView({
   const handleCreateAppointment = async () => {
     if (!appointmentFormRef.current) return [];
 
-    const values = appointmentFormRef.current;
+    const values = appointmentFormRef.current as unknown as FormValues;
 
     try {
-      // If client is specified, get client details for title
-      if (values.client) {
-        const response = await fetch(`/api/client?id=${values.client}`);
+      // If client is specified, get client group details for title
+      if (values.client && values.client.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)